src/decoder/image.rs

```diff
@@ -347,14 +347,19 @@ impl Image {
                     ),
                 )),
             },
-            PhotometricInterpretation::BlackIsZero | PhotometricInterpretation::WhiteIsZero
-                if self.samples == 1 =>
-            {
-                Ok(ColorType::Gray(self.bits_per_sample))
+            PhotometricInterpretation::BlackIsZero | PhotometricInterpretation::WhiteIsZero => {
+                match self.samples {
+                    1 => Ok(ColorType::Gray(self.bits_per_sample)),
+                    _ => Ok(ColorType::Multiband {
+                        bit_depth: self.bits_per_sample,
+                        num_samples: self.samples,
+                    }),
```

Comment:
Have kept the 1-band image as `ColorType::Gray` to preserve backwards compatibility as mentioned at https://github.com/image-rs/image-tiff/pull/216#discussion_r1506951296. TIFF files with bands>1 would use the new `ColorType::Multiband` enum variant here.